WordPress/gutenberg
# Fix: Error while Calling edit-site getCurrentTemplateTemplateParts selector

Fixes an issue correctly pointed out by @Mamaduka at https://github.com/WordPress/gutenberg/pull/62033/files#r1673431541.
We removed the editor private selector getCurrentTemplateTemplateParts but "edit-site" had an equivalent public selector that just called the editor private selector. Removing made the public selector throw an error. Checking the plugin repository this selector has some usage so we can not remove it right now.


## Testing Instructions
On the browser console, I pasted `await wp.data.select('core/edit-site').getCurrentTemplateTemplateParts();`.
I verified the selector worked as expected and there was a deprecation warning on the console.
